doc/operation-and-maintenance/Logging-&-monitoring.md

```diff
@@ -133,3 +141,15 @@ Wait a while, then open:
     http://172.17.0.2/render?from=-10mins&until=now&target=stats.example
 
 Then, if you configure your MongooseIM to send Exometer reports to that IP address and run it for a while, you should be able to see some interesting charts.
+
+### Run graphite in Docker - alternative method
+
+If one of these steps above doesn't work for you (e.g. they may be incompatible with your Docker on Mac), please try a less advanced alternative.
+It doesn't require docker machine and will expose Graphite ports on localhost, instead of using a new route.
```

Comment:
It doesn't require a docker machine and will expose Graphite ports on localhost, instead of using a new route.